PortfolioStyle.css

```diff
@@ -0,0 +1,379 @@
+	
+/*General CSS - normalize file contains most of it */
+	
+* {
+	box-sizing: border-box;
+}  
+	
+/* Using rems - set up to be 10px = 1 rem*/
+
+body {
+	font-size: 62.5%;
+	font-family: "Hind Guntur";
+	font-weight: 300;
+	color: #000500 ;
+	background-color: #edeef2;
+}
+/* Colours
+
+Background: #edeef2 ( White)
+Borders: #CED3DC (Grey)
+Black ish: #000500
+
+
+Hover color: #ABA9C3 (Lilac)
+Accents: #88292F (Red)
+
+*/
+
+
+
+
+/*Text*/
+
+h1 {
+	font-size: 5rem;
+	font-family: "Josefin Sans";
+	padding: 0 0 0 20px;
+}
+
+h2 {
+	font-size: 2rem;
+	text-align: center;
+	font-weight: 200;
+	padding: 50px 0 0 0;
+}
+
+h3 {
+	font-size: 1.5rem;
+}
+
+h4 {
+	font-size: 1rem;
+	font-weight: 200;
+}
+h6 {
+	font-size: 0.9rem;
+	padding: 20px 0 0 0;
+	font-weight: 100;
+	margin: 0;
+}
+
+p {
+	font-size: 1.2rem;
+	text-align: center;
+}
+
+
+/*Nav Bar*/
+
+nav {
+	overflow: hidden;
+	position:fixed;
+	display: flex;
+	top: 0;
+	width: 100%;
+	margin: 0;
+	padding: 0 0 5px 0;
+	font-size: 1.5rem;
+	background-color: #edeef2;
+}
+
+li {
+	display: inline;
+	list-style-type: none;
+	padding: 0 0 20px 0;
+
+}
+
+li a {
+	padding: 12px;
+	text-decoration: none;
+	color: #000500;
+	border-right: 1px solid #000500;
+}
+
+li a:hover {
+	background-color: #ABA9C3;
+}
+
+
+
+/*Body Styling*/
+
+.container {
+	display: flex;
+	height: 90vh;
+	justify-content: space-evenly;
+	flex-wrap: wrap;
+	align-items: center;
+	padding: 20px 20px;
+}
+
+.intropage .container {
+	height: 100vh;
+}
+
+.box {
+	text-align: center;
+	width: 30%;
+	height: 45%;
+	box-shadow: 0 5px 5px -5px #000500;
+	background-color: #edeef2;
+	overflow: hidden;
+	margin: 10px;
+	min-height: 350px;
+}
+
+.medbox {
+	text-align: center;
+	width: 40%;	
+	
+}
+
+.box img {
+  	display: block;
+  	margin-left: auto;
+  	margin-right: auto;
+  	object-fit: cover;
+	width: 100%;
+  	height: 25vh;
+  	background-color: #CED3DC;
+}
+
+
+.medbox img {
+  	display: block;
+    margin-left: auto;
+    margin-right: auto;
+	width: 100%;
+	height: 80vh;
+	object-fit: cover;
+    border-radius: 8px;
+    padding:10px;
+    border: 1px solid #CED3DC;
+}
+
+.aboutmebackground {
+	background-image: url(AboutMeBG3.jpg);
+	background-size: auto 100%;
+	background-repeat: no-repeat;
+	background-position: center;
+}
+
+.portfolio,
+footer {
+	background-color: #ABA9C3;
+}
+
+.container.project {
+	overflow: scroll;
+	object-fit: cover;
+	width: 100vw;
+	height: auto;
+}
+
+
+footer {	
+	text-align: center;
+	padding: 10px;
+	box-shadow: inset 0 0 10px #000500;
+}
+
+.social {
+	margin: 10px;
+}
+
+
+
+
+/* Social Buttons */
+
+ /* Style all font awesome icons */
+ 
+.fa {
+ 	padding: 20px;
+  	font-size: 1rem;
+  	width: 50px;
+  	text-align: center;
+  	text-decoration: none;
+	margin: 5px;
+}
+
+/* Add a hover effect */
+
+.fa:hover {
+ 	 opacity: 0.7;
+}
+
+/* Set a specific color for each brand */
+
+.fa-twitter {
+	background: #55ACEE;
+	color: white;
```

Comment:
please use hex for all colors where possible, generally never use color word values